docs/reference/config-options.md

```diff
@@ -143,16 +143,19 @@ The `conferenceCalling` section is optional in `featureFlags`. If it is omitted
 
 See also: conference falling for personal accounts (below).
 
-### File Sharing
+### File Sharing (optional)
 
-File sharing is enabled by default.  If you want to disable it for all teams, add this to your feature config settings:
+File sharing is enabled and unlocked by default. If you want to disable and/or lock it for all teams, add this to your feature config settings:
 
-```
+```yaml
 fileSharing:
   defaults:
-    status: enabled
+    status: disabled
+    lockStatus: locked
 ```
 
+`lockStatus` (required) can be either `unlocked` (feature status can be updated via team management) or `locked` (feature status can not be updated).
```

Comment:
The release in the changelog say that file sharing can be changed with an internal api endpoint. This sentence indicates that it's possible via team management (i.e. that a team owner/admin can set it). That doesn't sound coherent: either there is a public API where this can be set per team by a team admin/owner ("team management") or there is only an internal API in which case only the Wire customer support team can manually enable this on Wire.com.

Can you either update this sentence; or the release changelog file?

Reply:
I updated the changelog and docs. It should make things more clear now. Thanks for pointing this out.